src/pages/staff/[user_id].tsx

```diff
@@ -42,18 +43,20 @@ const ViewParaPage = () => {
         <b>Para Email:</b> {para?.email}
       </p>
 
-      <button
-        className={`${$button.default} ${$home.bold}`}
-        onClick={() => setUnassignParaPrompt(true)}
-      >
-        Unassign Para
-      </button>
+      {me?.user_id !== user_id && (
+        <button
+          className={`${$button.default} ${$home.bold}`}
+          onClick={() => setUnassignParaPrompt(true)}
+        >
+          Unassign Para
+        </button>
+      )}
 
       {unassignParaPrompt ? (
         <div>
           <p>
-            Are you sure you want to unassign {para?.first_name}{" "}
-            {para?.last_name}?
+            Are you sure you want to unassign
+            {para?.first_name} {para?.last_name}?
```

Comment:
This will have to be done with template literals inside of backticks if you want the space to be recognized without the ```{" "}```